doc/release-notes.md

```diff
@@ -108,6 +108,8 @@ will trigger BIP 125 (replace-by-fee) opt-in. (#11413)
 - The `getpeerinfo` RPC now returns a `connection_type` field. This indicates
   the type of connection established with the peer. It will return one of six
   options. For more information, see the `getpeerinfo` help documentation.
+  Please note that this string output is unlikely to be stable in upcoming
+  releases as we iterate to best capture connection behaviors.
```

Comment:
This is a generally correct statement, that could go into the RPC help instead?

Reply:
done